### PR TITLE
Expose Vault to PyHSPlasma

### PR DESCRIPTION
**Pull request by Hoikas**
As best I could tell, `plVaultStore` owns the nodes and generally speaking returns copies, so all the nodes types are value-wrapped.

### CODE REVIEW

File: Python/Vault/pyVaultNode.h [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+PY_WRAP_PLASMA_VALUE(VaultChronicleNode, plVaultChronicleNode);
+PY_WRAP_PLASMA_VALUE(VaultPlayerInfoListNode, plVaultPlayerInfoListNode);
+PY_WRAP_PLASMA_VALUE(VaultMarkerNode, plVaultMarkerNode);
+PY_WRAP_PLASMA_VALUE(VaultAgeInfoNode, plVaultAgeInfoNode);
+PY_WRAP_PLASMA_VALUE(VaultAgeInfoListNode, plVaultAgeInfoListNode);
+PY_WRAP_PLASMA_VALUE(VaultMarkerListNode, plVaultMarkerListNode);
+
+inline PyObject* pyPlasma_convert(const plVaultNode& node)
+{
+    if (plVaultPlayerNode* access = node.upcastToPlayerNode())
```
Comment (zrax) [inline Python/Vault/pyVaultNode.h]: Each of these upcast calls checks fNodeType independently, making this somewhat inefficient...  Why not use a switch instead?

Comment (Hoikas) [reply Python/Vault/pyVaultNode.h]: Good point. This was mostly me being lazy, not wanting to have to bang out the cast myself, due to the scope of the changes. I'll go back and hit this.

Comment (Hoikas) [reply Python/Vault/pyVaultNode.h]: Updated to fix this

**Comment by Hoikas** [comment]
:boom:

**Pull request merged**